AutomergeUniffi/automerge.swift

```diff
@@ -1495,6 +1527,96 @@ public func FfiConverterTypeSyncState_lower(_ value: SyncState) -> UnsafeMutable
     FfiConverterTypeSyncState.lower(value)
 }
 
+public struct Change {
+    public var actorId: ActorId
+    public var message: String?
+    public var deps: [ChangeHash]
+    public var timestamp: Int64
+    public var bytes: [UInt8]
+    public var hash: ChangeHash
+
+    // Default memberwise initializers are never public by default, so we
+    // declare one manually.
+    public init(
```

Comment:
Is there a reason to have this public? It feels like it should only ever be used internally.

Reply:
(I had the same question when I read through, but realized later that he set up a replica struct in Automerge with the same name and initializes it from this type, rather than passing through the type from AutomergeUniffi and adding extensions onto it to provide the additional equality and hashable conformances)

Reply:
Ah gotcha.